### PR TITLE
Bump aws-sdk-go version to v1.44.181

### PR DESCRIPTION
**Pull request by Vandita2020**
Issue #, if available:

Description of changes:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: test/e2e/tests/test_function.py [hunk 1]
```diff
@@ -137,7 +137,7 @@ def test_smoke(self, lambda_client):
         cr["spec"]["description"] = "Updated description"
         cr["spec"]["timeout"] = 10
         cr["spec"]["tags"] = update_tags
-
+
```
Comment (A-Hilaly) [inline test/e2e/tests/test_function.py:140]: Please remove these extra lines

**Review by A-Hilaly — APPROVED** [review]
Excellent! Thank you @Vandita2020

File: test/e2e/tests/test_function.py (hunk 1 above)
Comment (Vandita2020) [reply test/e2e/tests/test_function.py]: Done

**Review by A-Hilaly — APPROVED** [review]
/lgtm

**Pull request merged**